code/game/objects/items/toys.dm

```diff
@@ -1088,8 +1088,6 @@
 	item_state = "cultdagger"
 	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
-	inhand_x_dimension = 32
```

Comment:
These didn't need to be set. They were present on the original because their parent override them to 64x64, however that is not the case for item/toy